src/main/kotlin/com/kotlinerskt/cli/giskt/App.kt

```diff
@@ -1,5 +1,7 @@
 /*
  * This Kotlin source file was generated by the Gradle 'init' task.
+ * como olvidar cuando Gorro tuvo su momento de iluminacion y puso "null" en el live de hoy XD
+ *aguacate no malta XD
```

Comment:
IN THIS ORGANIZATION WE RESPECT SPACES!

```suggestion
 * aguacate no malta XD
```